packages/seacas/libraries/ioss/src/Ioss_Region.C

```diff
@@ -3053,4 +3040,55 @@ namespace Ioss {
       topologyObserver->reset_topology_modification();
     }
   }
+
+  std::string Region::get_internal_change_set_name() const
+  {
+    return get_database()->get_internal_change_set_name();
+  }
+
+  std::tuple<std::string, int, double> Region::locate_db_state(double targetTime) const
+  {
+    IfDatabaseExistsBehavior  ifDatabaseExists{Ioss::DB_OVERWRITE};
+    unsigned int              dbChangeCount{1};
+
+    DynamicTopologyFileControl fileControl(const_cast<Region*>(this), get_file_cyclic_count(),
+                                           ifDatabaseExists, dbChangeCount);
+
+    return fileControl.locate_db_state(targetTime);
+  }
+
+  std::tuple<std::string, int, double> Region::get_db_max_time() const
+  {
+    IOSS_FUNC_ENTER(m_);
+    auto db = get_database();
+    if (!db->is_input() && db->usage() != WRITE_RESULTS && db->usage() != WRITE_RESTART) {
+      return std::make_tuple(get_internal_change_set_name(), currentState, stateTimes[0]);
+    }
+
+    IfDatabaseExistsBehavior  ifDatabaseExists{Ioss::DB_OVERWRITE};
```

Comment:
These two lines don't do anything...

Reply:
Or, you should pass `Ioss::DB_OVERWRITE` in place of `ifDatabaseExists` and `1` in place of `dbChangeCount` ?

Not totally sure what is meant here.  Could also use `get_topology_change_count()` and `get_if_database_exists_behavior()`...

Reply:
`fileControl` modifies those values as return values ... if you want it to be clearer as return values, I can look into this.

Reply:
But they are local to the function.  Do you want the regions values to be changed, or what is the intent?